### ISSUE TITLE
Events page media kit 

### ISSUE THREAD

**Issue by shuchibooksmith** [sonnetmedia/brooklinebooksmith.com#94]
We'd like to create a page that we can send out to publishers about all the venues where we host events. It'd be similar to the booksmith schedule - so it's own web page but not necessarily accessible for the general customer browsing our website. 

It'll include venue information and photos. We have about six options for venues, so I'm not sure what would be best - one page with information on all six, or one page with general info with a link to another page with more information. Thoughts?

**Comment by shuchibooksmith** [comment]
i should have the information for this page later this week!

**Comment by budparr** [comment]
Okay. Just send me the copy. You can even just start a new "landing page" in the CMS, but I'm happy to do it for you.

**Comment by shuchibooksmith** [comment]
I started to add a new "landing page" but I wasn't really sure what that would do? Also this needs to exist separately from the site - as in customers shouldn't be able to navigate to it, it just needs to be a url that we can send specifically to publishers. Here is all of the text info. Each of these spaces will include between 3-7 photos.

The Booksmith Event Spaces & Partnerships

Events take place in our downstairs Readers and Writers Room, which can hold up to 150 people. The Coolidge Corner Theatre directly across the street is available to us on most weekdays for ticketed events with theatres seating 220 or 440 people. We also have relationships with larger venues in Boston ranging from 500 to 1,200+ seats. 

Events are promoted on social media (12,200 Twitter followers, 9,300 Facebook Likes, 2,450 Instagram followers), our weekly B-Mail newsletter (7,500 subscribers), our in-store events posters, on a sandwich board in front of the store, and on flyers with events listings placed in every customer's bag. Event books are always displayed in the front windows leading up to the event and if the publisher provides cover blow-ups we hang them over the front registers. We also send out a bi-monthly press release to media outlets and our events are listed in The Boston Globe and other publications. We are a New York Times reporting store.

Brookline Booksmith Readers and Writers Room (Downstairs)
Capacity Readings/Signing: 110-130 seated, plus standing room for 40+.
Recent Events: Cheryl Strayed, JoJo Moyes, Joe Hill, Mark Z. Danielewski, Colm Toibin, Jeanette Winterson, Alice Hoffman, Ha Jin.
Recent Success Notes: This past fall we hosted four local launches with crowds of 120+ and sales of 50+ frontlist copies. Our biggest in-store event of the fall was Colleen Hoover who drew a crowd of approximately 140 with 148 frontlist sales.

Brookline Booksmith Store Level (Upstairs Signing Only)
Capacity Signing Only: 750 best to date.
Recent Events: Mindy Kaling, Neil Patrick Harris, Grace Helbig, Joey Graceffa, Bill Nye, Sara Bareilles, Ree Drummond.
Recent Success Notes: The biggest signing-only event we’ve hosted was 750 people for Youtuber Joey Graceffa. Our other biggest signing-only events had attendance capped by the publisher. These include Neil Patrick Harris (600 books in under three hours), Jeff Kinney (500 people with kids activity stations) and Mindy Kaling (400 people in under two hours). Staff members are always available to take pictures, but we also have a great relationship with a professional photographer who can take customers’ pictures and upload them to a website immediately after the event. We’ve perfected our setup for a standard signing line and have a wristband system in place for distributing pre-signed books. 

Coolidge Corner Theatre Event Space
Capacity: Moviehouse 1 can hold 440; Moviehouse 2 can hold 220.
Recent Events: Michael Chabon, Jhumpa Lahiri, Richard Ford, Junot Diaz, Felicia Day, John Waters, Caroli
ne Kennedy, Judy Blume, Brandon Sanderson, Christopher Moore, Joe Perry of Aerosmith.
Recent Success Notes:  Coolidge events usually have sales of 200+ frontlist units for Moviehouse 1 and 100+ for Moviehouse 2. In the past couple of months we’ve had sold-out events for Brandon Sanderson and Felicia Day, both of whom walked the line outside of the theater, greeting fans and generating terrific energy throughout the neighborhood. They also had exemplary sales with 410 hardcovers sold for Felicia Day (MH1) and 211 hardcovers sold for Brandon Sanderson (MH2). After our Coolidge event with Joe Perry of Aerosmith, his manager told me, “You ran the best event of the tour with the worst people….” (meaning troublesome fans, not us). The Coolidge is directly across the street from the store and post-Coolidge signings are held at Brookline Booksmith. The signings are open to all, giving us the opportunity to sell books to non-ticketholders if the theater is sold-out.

Berklee Performance Center Event Space
Capacity: 1,215 (732 Orchestra Level + 432 Balcony Level).
Recent Events: Elvis Costello in conversation with Peter Guralnick.
Recent Success Notes: We sold 900 ticket/book bundles for Elvis Costello’s memoir in October. The Performance Center is a beautifully designed space with terrific acoustics. From the back row of the top balcony you still have a great view of the stage and can hear every word. Their technical setup is extraordinary and they make a great partner for promoting music-related books. Beyond the Performance Center, Berklee has several smaller rooms and will soon make a 500-seat theater available to us. 

Wilbur Theatre Event Space
Capacity: 1,200.
Recent Events: Lena Dunham, Nick Offerman, Rainn Wilson.
Recent Success Notes: The Wilbur is very collaborative in their marketing strategies and are eager to use their connections as Boston’s preeminent comedy theatre to promote our events. Beyond our events being promoted alongside theirs on their website and print advertising, they’ve gone so far as to feature our events on digital billboards along the highway, and use their marketing team to help coordinate radio interviews and promotional giveaways.

Congregation Kehillath Israel
Capacity: 100-1,000.
Past Events: Jonathan Safran Foer, Amy Tan and many more.
Recent Notes: We have not had occasion to use this space in the past year, but I recently met with their rabbi and he is eager to resume our partnership. There are three events spaces: a chapel that seats 100; an open room with a stage that can seat 200; and the main worship space which seats 400-500 and can be arranged to seat up to 1,000 if the balcony is used.

**Comment by shuchibooksmith** [comment]
I now have all the photos accessible via google drive, what's the best way for me to get them to you?

**Comment by budparr** [comment]
Have you cropped and compressed the images? https://github.com/sonnetmedia/brooklinebooksmith.com/wiki/image-compression

**Comment by zchyde** [comment]
Shuchi I can help you with this, give me access to the photos

**Comment by shuchibooksmith** [comment]
@zchyde shared it with you via google, thanks!

**Comment by zchyde** [comment]
Need a little bit more information for this - Events wants these photos in carosels, with copy in between. So images for our events wiht the Coolidge Corner theatre will be above or below a section called "The Coolidge Corner Theatre" with information about the theatre, what our events there are like, et al.

What dimensions would you suggest for a carosel like this? what are the height dimensions on the carosels you have on kids + home? I checked the wiki but I don't think that info was there. Thoughts?

**Comment by peterbooksmith** [comment]
@budparr do you think it's possible to have a draft of this ready by March 8 and to have a final version done on March 11?  Asking because our events manager will be meeting with publicists the following week and would be very good to have it ready for him.

**Comment by budparr** [comment]
Absolutely. @zchyde are those questions there for me?

**Comment by zchyde** [comment]
yessir. I just need dimensions to resize the pictures to if they're all going in those carosels.

**Comment by budparr** [comment]
@zchyde IMHO we don't use a carousel, but make this essentially like the [holiday gift guide](http://www.brooklinebooksmith.com/holiday-gift-guide/2015/12/22/holiday-gift-guide-mindfulness-and-home/). I'd make the images a bit bigger than we need them, but not too big (say max 500px wide). What do you all think?

**Comment by peterbooksmith** [comment]
@budparr  - sorry, were you waiting on more feedback from us here?  I am personally inclined to go with your idea on this.  The events dept (Tom and Tim) may say something different when they see a draft.  FYI - the carousel idea has popped up more than once around here for this and other things.  Not sure why it seems to be popular with some people, but it may come up again.

**Comment by zchyde** [comment]
Nope, we're waiting on me for this. Working on it. 

I think the carosel idea is so popular because its a way of displaying a lot of imagery in a unique way, however I think this is one of those moments you and I and Shuchi discussed ,where Bud is often cautioning us against them because they tend to be an overimplimented web item that people rarely use, but our staff doesn't seem to retain or believe that information. I could be wrong.

**Comment by peterbooksmith** [comment]
That sounds about right, I prob just didn't retain that information.  Will be helpful when I have to explain to others why their carousel idea might not be the best option.

On this @zchyde - is it progressing so that there will be something for Tom & Tim to see later tomorrow?  Just checking.

**Comment by zchyde** [comment]
I'm guessing not, what I'm doing is compressing these images (500 of them) one at a time so that we can upload them to the website

**Comment by shuchibooksmith** [comment]
i think that's true. i like bud's idea of displaying images similar to 
how we displayed them in the gift guide. but because these are photos of 
a space (and i imagine people will want to look at the full size) is 
there a way we can do that @budparr?

On 3/7/2016 5:09 PM, Zoe wrote:
>
> Nope, we're waiting on me for this. Working on it.
>
> I think the carosel idea is so popular because its a way of displaying 
> a lot of imagery in a unique way, however I think this is one of those 
> moments you and I and Shuchi discussed ,where Bud is often cautioning 
> us against them because they tend to be an overimplimented web item 
> that people rarely use, but our staff doesn't seem to retain or 
> believe that information. I could be wrong.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/sonnetmedia/brooklinebooksmith.com/issues/94#issuecomment-193478091>.
>

-- 
Shuchi Saraswat
Backlist Buyer
Brookline Booksmith
279 Harvard Street
Brookline, Massachusetts 02446
617.566.6660
www.brooklinebooksmith.com

P Please consider the environment before printing this email.

**Comment by zchyde** [comment]
I wrote a comment here that didnt belong here sorry!

**Comment by zchyde** [comment]
I dont think Tim and Tom would be able to see anything tomorrow, the find of the week has exclipsed my work on this. Im swtiching gears now.

**Comment by peterbooksmith** [comment]
Ok, sorry I didn't chime in earlier.  I know find of the week has been on our lists for a while, but this def comes first, as they want to have it finished by the end of the week.  If anyone else in the store wants to interrupt you with other work Zoe, please explain this is your priority right now and direct them to me if needed.

**Comment by zchyde** [comment]
ok, I think I can get these done before I leave.

**Comment by peterbooksmith** [comment]
thanks - I should have stated this earlier, made sure you were aware

**Comment by zchyde** [comment]
ok, resized and compressed images are in our dropbox 

https://www.dropbox.com/sh/rizv73mvnr5s6bk/AADtGeglw0TlGTmwIp610iz0a?dl=0

**Comment by budparr** [comment]
I'll have a prototype for you by tomorrow.

**Comment by budparr** [comment]
Here's the page: http://www.brooklinebooksmith.com/the-booksmith-event-spaces--partnerships/

At this moment, I put this up just like you would with a gift guide page. I think the images are pretty good sized. I can link to larger images (though they're not much larger than what's displayed on the page) with a modal (pop-up thingy), though think that might just add weight to the page and not much value to the users. But let me know what you want.

**Comment by zchyde** [comment]
Looks like the events are all on there twice each, is this On Purpose?

**Comment by budparr** [comment]
Oops. Fixed.

**Comment by zchyde** [comment]
Excellent, thank you. I'm going to send this up the chain, it seems like theres a little photo weirdness at the bottom there as well, Lena Dunham is just sort of floating.

**Comment by budparr** [comment]
The images would work best as uniform sizes. They float to the left so they appear inline and when they're different sizes they can get a bit wonkey. The easiest fix might just be to change the order of them (which I think you'll find easier to do on Github than in the editor, @zchyde ).

The images are stored here: 
https://console.cloud.google.com/storage/browser/images.brooklinebooksmith.com/event-venues/?project=brookline-booksmith&authuser=0

**Comment by zchyde** [comment]
Ok, I will tinker with it, thank you

**Comment by peterbooksmith** [comment]
@budparr Tom & Tim sent us their feedback, and I can pass all of that along. A lot of is changes to text, some spacing stuff, but they did ask for the photos to be in a carousel.  :)   I would like to discuss that with them tomorrow.  If that is the way they absolutely want to go, I assume that should be dealt with first because it affects the rest of the design of the page?  Then text changes can be done?  If you would rather just see what they said first, just let me know.

**Comment by budparr** [comment]
Thanks, @peterbooksmith  - I cleaned up the copy just a bit (I emphasized the section headers) and I moved one of the images up with the copy, which I think helps. So have a look at that. 

I understand the need to have the page a bit more uniform or neat looking, so the desire for a carousel is understandable. Having the images on the page like this is a bit more utilitarian, but it does ensure people will see them (carousel almost ensures that some of them will be ignored (according to every study I've seen)), so it depends on priorities. 

Also, one other, though secondary, consideration, is that as it stands this is merely a page that you can go into the editor and change, and if add carousels it becomes a page that is coded. 

No problem either way. Any content changes can be changed by Zoe in the editor.

**Comment by peterbooksmith** [comment]
Thanks - I am going to discuss this with Tim & Tom in a couple of hours, will get back to you after that.

**Comment by peterbooksmith** [comment]
Carousel - that is the most concise answer.  That's that they really want and this page is really just for their purposes and won't be seen or used by the general public, so ok.  They would also like the images in the carousel to get embiggened when one hovers over them with the cursor.  Does that make sense?  I know that makes for a lot of stuff going on, but that's what they really want.

**Comment by peterbooksmith** [comment]
With this addition just received:

Hi Peter,
 
It looks like Bud has already fixed some of the issues from yesterday and it’s looking great. Could we keep the format he implemented with the first image to the left of the text for each venue and the remaining images on a carousel below?
 
A top banner image and text edits (which will be minimal) coming soon.
 
Thanks!
 
-Tom

**Comment by budparr** [comment]
Sounds good. I'll start working on the carousels

**Comment by budparr** [comment]
I put the carousel in. As I mentioned earlier, you probably shouldn't edit the page now in the CMS editor (you could, but I think there's a good chance it wouldn't work out well), but @zchyde can edit it right here on Github: https://github.com/sonnetmedia/brooklinebooksmith.com/blob/gh-pages/_landing_pages/the-booksmith-event-spaces--partnerships.md

Right now the images don't enlarge or anything like that. I'll have to look at some different technology solutions for that, so that could take a few days.

just a reminder of the link: http://www.brooklinebooksmith.com/the-booksmith-event-spaces--partnerships/

**Comment by peterbooksmith** [comment]
Thanks Bud.  @zchyde - are you able to get to work on Tom's word/text changes, aim to have that done sometime tomorrow?

**Comment by zchyde** [comment]
@peterbooksmith yep. Did we get a banner image from them or should I just put in a placeholder?

**Comment by zchyde** [comment]
Oh! I have to say, @budparr , this is pretty cool with the carosels. I think this will work well for this page.

**Comment by peterbooksmith** [comment]
No banner image yet, that is "coming soon" per Tom.  I have a collection of his requested text edits, I can hand that to you so that you have it all in one place.

**Comment by budparr** [comment]
Thank you. I think you're right. Despite my (and [others](http://shouldiuseacarousel.com/)) caution about carousels, I do think it works here, and hopefully this audience will be motivated enough to flip through the images.

**Comment by budparr** [comment]
@zchyde when you make edits to the copy in Github, if you can, when it asks you to explain what you're doing (box under "commit changes"), just write in #94 and the change will also be registered here with this issue.

**Comment by shuchibooksmith** [comment]
this whole page looks GREAT. thank you all for handling!

On 3/9/2016 5:26 PM, Zoe wrote:
>
> Oh! I have to say, @budparr <https://github.com/budparr> , this is 
> pretty cool with the carosels. I think this will work well for this page.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/sonnetmedia/brooklinebooksmith.com/issues/94#issuecomment-194539110>.
>

-- 
Shuchi Saraswat
Backlist Buyer
Brookline Booksmith
279 Harvard Street
Brookline, Massachusetts 02446
617.566.6660
www.brooklinebooksmith.com

P Please consider the environment before printing this email.

**Comment by zchyde** [comment]
@budparr ok. Something you want me to do every time i commit?

**Comment by zchyde** [comment]
oh whoops, ok I get how it works now

**Comment by zchyde** [comment]
Ok, I believe all of Toms changes (banner excluded) have been made, @peterbooksmith

**Comment by budparr** [comment]
If you're in the CMS/editor it makes the commit messages for you, of course. So no worries there. Otherwise, it's good to give some details about what you're doing, including, if applicable, reference the issue the change relates to.

**Comment by budparr** [comment]
@zchyde here's the interesting part. If you click on your commit here in this issue (it's the "e041317" to the right of where it says "…Zchyde added a commit…" you'll see exactly what you changed, which is the point of making the references. We could even discuss those changes inline, etc.

**Comment by zchyde** [comment]
@budparr  it would appear I have ruined everything? 

http://www.brooklinebooksmith.com/the-booksmith-event-spaces--partnerships/

**Comment by zchyde** [comment]
And in the CMS!

**Comment by budparr** [comment]
You can't edit this page in the CMS. See note above.

**Comment by zchyde** [comment]
oh I thought that just mean I can't edit the carousels, I'm sorry

**Comment by budparr** [comment]
Stand by. I'll fix it.

**Comment by budparr** [comment]
Sorry that took a while. All three of my kids are home sick from school, so I'm a bit scattered. 

The beauty of having everything in version control is that it's easy to revert changes back to an earlier version. So that's done. Just make your changes for this page right in Github. Let me know if you have any problems with that.

**Comment by budparr** [comment]
If you give me a few minutes, I see if I can make this editable in the CMS for you.

**Comment by zchyde** [comment]
Oh, my god I'm so sorry, thats too many sick kids. 

I don't mind editing it here in github, I just forgot/misunderstood. Although I do have a question, they want the banner to remain a decent size but don't want text on it. Another way of saying this is they want to get rid of text on the banner, which is easy to do - I just deleted it - but then the banner size shrinks to a mini banner, and I think they will want the full size. Is there a way I can keep the banner size without adding a title?

**Comment by zchyde** [comment]
I can take some screenshots if that made no sense.

**Comment by budparr** [comment]
Ah, yes. Once you're editing on Github you get to access secret fields!

Up in the front matter (where there's stuff like "title: this is a title") you can add a new line like this:

`banner_height: 350`

where, in this case, the banner height will be 350px. You can change "350" to any number you like.

**Comment by zchyde** [comment]
Peeeeeeerfect

**Comment by budparr** [comment]
As I slowly unveil to you the power of our publishing system… :sunglasses:

**Comment by peterbooksmith** [comment]
This still is not appearing correctly for me.  I think "Brookline Booksmith Programs & Partnerships" is supposed to be under the banner photo, but above the first paragraph that begins, "In-Store events...."?

**Comment by budparr** [comment]
@peterbooksmith I changed it. Let me know if that's how you want it. Have a great weekend!

**Comment by peterbooksmith** [comment]
@budparr - the font and/or size of that "Brookline Booksmith Programs & Partnerships" appears to be different than the other section headers below it.  I think that is the only thing.  Thanks.

**Comment by zchyde** [comment]
@peterbooksmith all set

**Comment by budparr** [comment]
Hi @peterbooksmith - that's by design. There's a hierarchy to headings. That one introduces the page, the ones below are section headers. I can change it if you like, but wanted to let you know that there's a purpose for it. Just let me know.

**Comment by budparr** [comment]
crossed paths with you @zchyde, thanks for making the change.

**Comment by zchyde** [comment]
@budparr I was too quick. Right now all the headers are the same size, I kind of don't have any feelings about it either way, I can change it back if we want.

**Comment by budparr** [comment]
I think I see the issue. I hadn't initially intended to use the "brand font" for headings, but had on the lower level ones. I fixed the inconsistency and put the higher level heading back as it was. Now we have the hierarchy and they look consistent.

**Comment by zchyde** [comment]
Oh excellent, I love it, perfect

**Comment by peterbooksmith** [comment]
Got it thanks. Yes, when last I had looked that top one looked like a slightly different font, and I see whaty you mean about the hierarchy.  I will double-check with Tim/Tom to make sure all is well with them, and then this should be done.

**Comment by peterbooksmith** [comment]
At some point, they may want to change some text or possibly add a map or more info about the store, but they are all set for now.

**Comment by budparr** [comment]
Great. I'll close the issue for now. Feel free to open a new issue in the future when they want changes.
